src/helpers/error.sender.js

```diff
@@ -2,17 +2,18 @@
  * A function to create an error object and use the the
  * res object to send it back to the client
  * @param {Number} status - The status code to send back
- * @param {object} resultObj - The result object to structure
- * before to send back
  * @param {Object} res - The res object
+ * @param {String} fileld - the invalid filed rasing the error
```

Comment:
@nkpremices do you mean `field - the invalid field raising the error`?

Also, check other instances of `fileld` and correct spelling.

Reply:
Oh no!! I've been avoiding this error hardly.
It won't happen again. 
I promise.

Reply:
Let me correct it then check other instances